src/main/java/gov/nasa/pds/validate/ValidateLauncher.java

```diff
@@ -306,13 +306,15 @@ public CommandLine parse(String[] args) throws ParseException {
    * @throws Exception If an error occurred while processing the command-line options.
    */
   public void query(CommandLine line) throws Exception {
+	int count = 0, every = Integer.valueOf(line.getOptionValue("everyN", "1")).intValue();
     List<Option> processedOptions = Arrays.asList(line.getOptions());
     List<String> targetList = new ArrayList<>();
     // Gets the implicit targets
     for (java.util.Iterator<String> i = line.getArgList().iterator(); i.hasNext();) {
       String[] values = i.next().split(",");
       for (String value : values) {
-        targetList.add(value.trim());
+        if ((count % every) == 0) targetList.add(value.trim());
+        count++;
```

Comment:
Inserting blindly is not very efficient. In my example I use success/b.xml twice and the same file is checked twice. It would be better to get absolute or canonical name and then insert it into a set rather than list. Then reduce the set after all inserts. It prevents processing same file twice and randomizes the every Nth processed.